extensions/interactions/interaction_specs.json

```diff
@@ -409,7 +409,16 @@
     "is_linear": false,
     "display_mode": "inline",
     "id": "SetInput",
-    "customization_arg_specs": [],
+    "customization_arg_specs": [
+      {
+        "description": "Button label",
```

Comment:
Be more explicit, here and in the .py file: "Label for the 'Add Item' button".

Reply:
Done. Thanks!